Vagrantfile

```diff
@@ -10,4 +10,5 @@ Vagrant.configure("2") do |config|
   config.vm.provision :shell, path: "vagrant/provision.sh"
   config.vm.provision :reload
   config.vm.provision :shell, path: "vagrant/build.sh", run: 'always', privileged: false
+  config.ssh.extra_args = "-t"
```

Comment:
maestro requires a TTY to run when logging is enabled. When trying to do:
```
vagrant ssh -c "sudo maestro"
```
maestro would fail because the logging utility could not bind a TTY.

By providing `-t` to the ssh command, it allocates a TTY to the session and I am able to run maestro in my system tests via ssh